essentials-framework/Essentials DM/Essentials_DM/AirMedia/AirMediaController.cs

```diff
@@ -263,6 +263,29 @@ public int NumberOfComPorts
         #endregion
 
 
+
+        #region IRoutingNumeric Members
+
+        public void ExecuteNumericSwitch(ushort input, ushort output, eRoutingSignalType signalType)
+        {
+

```

Comment:
I would add a check here in case the cast to `AmX00DisplayControl.eAirMediaX00VideoSource` fails. You can use `Enum.IsDefined(typeof(...), input)` to check if the input value is in the enum.

Reply:
Done!